to.us.scrollprotocol.Profectus.yml

```diff
@@ -0,0 +1,66 @@
+id: to.us.scrollprotocol.Profectus
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: profectus
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to talk to the network:
+  - --share=network
+  # Needs to save files locally
+  - --filesystem=xdg-download:rw
+  # Input
+  - --device=input
```

Comment:
`device=input` this will not work, as the necessary version of flatpak that know about this option is barely available, and not even from the stable branch.